### DISCUSSION TITLE
Neptune T10 Pointer blocks view of digits

### DISCUSSION THREAD

**Discussion by beyondunreal79** [jomjol/AI-on-the-edge-device#2682] [Q&A]
I feel I have a confusing meter.  I have a neptune T-10 water meter that reads in gallons. Its both digital and analog. The last digit on the digital part moves up every 10 gallons.  The analog part is a needle that rotates around the entire meter and reads gallons. since the red needle periodically covers the digital, I get false readings.  Is it possible for me just to use the data from the analog needle (which is always right on) and have it keep counting up? It counts up until it hits 0. Is this possible?

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/45919008/bca3d04e-1fdf-487a-b46f-27fe41972a98)

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/45919008/a6139221-2bbe-42a0-b2c5-83c6daa34af5)

**Comment by caco3** [comment]
> Its both digital and analog. 

You mean it has digits, not digital :)

The covering of the digits by the pointer is an issue.
This will lead to wrong readings.
You could try to use the https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-numbermaxratevalue and https://jomjol.github.io/AI-on-the-edge-device-docs/Correction%20Algorithm/#allownegativerates to catch wrong readings.
The meter then would report errors while the digits are covered and stay on the last valid value. Once the pointer moves off the digits, the error will go away and a valid reading should happen again, causing a little jump in your recordings.

If just using the pointer, you would need some external functionality to sum it up.

Also you have a strong reflection which also would need to be soled. You could try to use an external flash light for this (see the docs in https://jomjol.github.io/AI-on-the-edge-device-docs/External-LED/)

**Reply by beyondunreal79** [reply]
Believe it or not, that reflection has no bad effect on readings. I have been watching it carefully.  That pointer needle is a killer. If it weren't for that, it would be correct probably 99 percent of the time. Amazing project. Awesome work!

**Reply by beyondunreal79** [reply]
I was able to get home assistant to sum up just using the pointer. It kinda worked, but the issues was when it went from 9 to 0 (which should be 10) It didn't count anything until it hit 1. So I would lose 1/10 gallon with every 10 gallons. Then it starts to get messy.

**Reply by caco3** [reply]
I think you will need a template sensor which converts the 0 to a 10.
